draft-ietf-quic-multipath.md

```diff
@@ -319,6 +319,13 @@ to migrate to that path.  Instead, servers SHOULD consider new paths
 over which non-probing packets have been received as available
 for transmission.
 
+As specified in {{Section 9.3 of QUIC-TRANSPORT}}, the server SHOULD send a new
```

Comment:
```suggestion
As specified in {{Section 9.3 of QUIC-TRANSPORT}}, the server is expected send a new
```
Given there is already normative language in RFC9000 we should not repeat it here.